topbeat/beat/sigar.go

```diff
@@ -15,113 +15,41 @@ type SystemLoad struct {
 }
 
 type CpuTimes struct {
-	User          uint64  `json:"user"`
+	Cpu           sigar.Cpu
```

Comment:
The idea id not to copy each field from sigar.Cpu to the CpuTimes and copy all the structure at once. 

For example here:

	return &CpuTimes{
 -		User:    cpu.User,
 -		Nice:    cpu.Nice,
 -		System:  cpu.Sys,
 -		Idle:    cpu.Idle,
 -		IOWait:  cpu.Wait,
 -		Irq:     cpu.Irq,
 -		SoftIrq: cpu.SoftIrq,
 -		Steal:   cpu.Stolen,
 -	}, nil
 +	return &CpuTimes{Cpu: cpu}, nil